source/_integrations/xbox.markdown

```diff
@@ -267,7 +267,7 @@ Just like the binary sensors, the Xbox sensor platform automatically keeps track
 | `sensor.{gamertag}_account_tier` | Disabled | Shows your friend's Xbox Live account tier (Gold or Silver).             |
 | `sensor.{gamertag}_gold_tenure`  | Disabled | Shows how long your friend has had Xbox Live Gold.                       |
 
-## Media Source
+## Media source
 
 The Xbox media source platform allows you to use the Media Browser panel to view both your own, and community, gameclips or screenshots for games that you have installed on any of your consoles. As with any other media source integration, you are also able to send these clips to supported media players like Chromecast.
```

Comment:
```suggestion
The Xbox media source platform allows you to use the media browser panel to view both your own, and community, gameclips or screenshots for games that you have installed on any of your consoles. As with any other media source integration, you are also able to send these clips to supported media players like Chromecast.
```
"the media browser panel" could possible be a my link as well.
https://my.home-assistant.io/redirect/media_browser/